bindings/wasm/bindings.js

```diff
@@ -87,12 +87,12 @@ Module.setup = function() {
 
   const CrossSectionCtor = Module.CrossSection;
 
-  function cross(polygons, fillrule = 'Positive') {
+  function cross(polygons, fillRule = 'Positive') {
     if (polygons instanceof CrossSectionCtor) {
       return polygons;
     } else {
       const polygonsVec = polygons2vec(polygons);
-      const cs = new CrossSectionCtor(polygonsVec, fillrule = fillrule);
+      const cs = new CrossSectionCtor(polygonsVec, fillRule = fillRule);
```

Comment:
I did it when I thought that keywords existed (rather than just ordering like in cpp) out of python habit (in ocaml there is name punning, so wouldn't be doing the repetition there). I'll go back and drop it.